codelists/sustainabilityGoal.csv

```diff
@@ -1,15 +1,30 @@
 Code,Title,Description
 economic,Economic,"The goal is economic, such as formal employment and the local economy."
 economic.formalEmploymentPromotion,Formal employment promotion,The goal is to promote formal employment.
+economic.innovativePurchase,Innovative purchase,"The goal is to procure innovative goods, services or works."
 economic.localEconomyPromotion,Local economy promotion,The goal is to promote the local economy.
 economic.marketInnovationPromotion,Market innovation promotion,The goal is to promote market innovation.
+economic.processInnovation,Process innovation,"The goal is to procure goods, services or works that entail process innovation."
+economic.productInnovation,Product innocation,"The goal is to procure goods, services or works that entail product innovation."
```

Comment:
To match marketInnovationPromotion. Another option is to remove "Promotion" from existing codes (4x).

```suggestion
economic.processInnovationPromotion,Process innovation promotion,"The goal is to procure goods, services or works that entail process innovation."
economic.productInnovationPromotion,Product innovation promotion,"The goal is to procure goods, services or works that entail product innovation."
```